### PR TITLE
Add subscribers number limits

### PR DESCRIPTION
**Pull request by feymartynov**

### CODE REVIEW

File: docs/src/api/room.md [hunk 1]
```diff
@@ -2,10 +2,11 @@
...
-created_at     |        int | _required_ | Room creation timestamp in seconds.
-backend        |     string | _required_ | Room backend, either `janus` or `none`.
+Name              | Type       | Default    | Description
+----------------- | ---------- | ---------- | ----------------------------------------------------
+id                |       uuid | _required_ | The room identifier.
+audience          |     string | _required_ | The audience of the room.
+time              | [int, int] | _required_ | Opening and closing timestamps in seconds.
+created_at        |        int | _required_ | Room creation timestamp in seconds.
+backend           |     string | _required_ | Room backend, either `janus` or `none`.
+subscribers_limit |        int | _optional_ | The maximum number of simultaneously entered subscribers allowed.
```
Comment (khodzha) [inline docs/src/api/room.md:12]: simultaneously present?

Comment (manifest) [reply docs/src/api/room.md:12]: Or maybe just (room)`size`, or (room) `size_constraint`, or a nested dictionary – `constraint.size` ?

File: src/app/endpoint/room.rs [hunk 2]
```diff
@@ -293,12 +299,29 @@ impl RequestHandler for EnterHandler {
...
+            // Find opened room.
+            let room = db::room::FindQuery::new()
                 .id(payload.id)
                 .time(db::room::now())
                 .execute(&conn)?
                 .ok_or_else(|| format!("Room not found or closed, id = '{}'", payload.id))
-                .status(ResponseStatus::NOT_FOUND)?
+                .status(ResponseStatus::NOT_FOUND)?;
+
+            // Check if not full house in the room.
```
Comment (manifest) [inline src/app/endpoint/room.rs:310]: Quite a strange comment.

File: docs/src/api/room.md (hunk 1 above)
Comment (manifest) [reply docs/src/api/room.md:12]: We also don't use a term `subscriber` for video conferences. We use `agent`, `publisher`, `listener`, etc.

Comment (alexkonst) [reply docs/src/api/room.md:12]: `capacity`?

Comment (feymartynov) [reply docs/src/api/room.md:12]: > We also don't use a term subscriber for video conferences. We use agent, publisher, listener, etc.
[We do](https://github.com/netology-group/janus-conference/search?q=subscribers&unscoped_q=subscribers) actually.

File: src/schema.rs [hunk 3]
```diff
@@ -32,6 +32,7 @@ table! {
         handle_id -> Int8,
         session_id -> Int8,
         created_at -> Timestamptz,
+        subscribers_limit -> Nullable<Int4>,
```
Comment (manifest) [inline src/schema.rs:35]: Do we really need `Nullable` here?

Comment (feymartynov) [reply src/schema.rs:35]: I think it would better be an optional feature.

File: src/app/endpoint/room.rs (hunk 2 above)
Comment (feymartynov) [reply src/app/endpoint/room.rs:310]: ¯\_(ツ)_/¯

**Pull request merged**